blockchain/message_pool/src/msgpool/msg_pool.rs

```diff
@@ -391,9 +391,9 @@ where
         Ok(publish)
     }
 
-    /// Finish verifying signed message before adding it to the pending mset hashmap. If an entry
-    /// in the hashmap does not yet exist, create a new mset that will correspond to the from
-    /// message and push it to the pending hashmap.
+    /// Finish verifying signed message before adding it to the pending `mset` hash-map. If an entry
```

Comment:
`hashmap` should be fine, no? https://en.wiktionary.org/wiki/hashmap

Reply:
We can add it to the custom dictionary.